src/main/java/lotto/Main.java

```diff
@@ -0,0 +1,29 @@
+package lotto;
+
+import lotto.domain.Lotto;
+import lotto.domain.LottoChecker;
+import lotto.domain.LottoGenerator;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Main {
+    public static void main(String[] args) {
+        long price = InputView.initLottoPrice();
+        LottoGenerator lottoGenerator = new LottoGenerator(price);
+        ResultView.printLottos(lottoGenerator.getLottos());
```

Comment:
```suggestion
        List<Lotto> lottos = LottoGenerator.generate(price);
        ResultView.printLottos(lottos);
```
LottoGenerator가 상태값을 가지기 보다 위와 같이 클래스 메서드로 구현하는 것은 어떨까?
AI와의 피드백을 통해 각각의 방식으로 구현할 때의 장단점을 비교해 보고 리팩터링해본다.